packages/ui5-task-stringreplacer/README.md

```diff
@@ -49,13 +52,25 @@ builder:
     - name: ui5-task-stringreplacer
       afterTask: replaceVersion
       configuration:
-        files: ["**/*.js", "**/*.xml"]
+        files:
+          - "**/*.js"
+          - "**/*.xml"
+      replace:
+        - placeholder: ${project.artifactId}
+          value: my.sample.app
+        - placeholder: ${project.version}
+          value: 1.0.0-SNAPSHOT
+        - placeholder: '{{app.AppTitle}}'
+          value: My Sample App
 ```
+
 3. Maintain all string placeholders and values in `.env` file
```

Comment:
same comment as for the middleware - not sure how this fits in addition to the config in 2.

Reply:
Both configuration options are possible at the same time and will be merged. You can either use the `.env` or the `ui5.yaml` to provide placeholder values.